src/widget/templates/detail.hbs

```diff
@@ -77,18 +77,18 @@
             {{#if actionItems}}
             <div class="buttonItem">
                 <button id="directionsBtn">
-                    Get Directions
+                    {{dirBtnText}}
                 </button>
             </div>
             <div class="buttonItem">
                 <button id="contactBtn">
-                    Contact
+                    {{contactBtnText}}
                 </button>
             </div>
             {{else}}
             <div class="buttonItem long">
                 <button id="directionsBtn">
-                    Get Directions
+                    {{dirBtnText}}
```

Comment:
Did we try to do this with bfString attribute? You can check out how its implemented in other plugins through this link: https://github.com/search?q=org%3ABuildFire+in%3Acode+bfstring&type=code

Reply:
I did try it this way but couldn't make it work, so I got the string data in the details controller and passed them here

`dirBtnText: strings.get("ActionButtons.directionsButton").substring(0, 14) +  "..." || "Get Directions"`